### PR TITLE
code quality

### PR DESCRIPTION
**Pull request by klodav**

### CODE REVIEW

File: app/grails-app/services/de/laser/LicenseService.groovy [hunk 1]
```diff
@@ -12,8 +12,8 @@ class LicenseService {
     AccessService accessService
     ContextService contextService
 
-    List getMyLicenses_readRights(){
-        List result = []
+    List<License> getMyLicenses_readRights(){
+        List<License> result = []
         List tmpQ
```
Comment (croqueGrec09) [inline app/grails-app/services/de/laser/LicenseService.groovy:17]: Type of tmpQ?

File: app/grails-app/services/de/laser/PropertyService.groovy [hunk 2]
```diff
@@ -111,9 +110,9 @@ class PropertyService {
         [query: base_qry, queryParams: base_qry_params]
     }
 
-    def getUsageDetails() {
+    List getUsageDetails() {
         List usedPdList  = []
```
Comment (croqueGrec09) [inline app/grails-app/services/de/laser/PropertyService.groovy:114]: Type of List?

File: app/grails-app/services/de/laser/PropertyService.groovy [hunk 3]
```diff
@@ -111,9 +110,9 @@ class PropertyService {
...
-        Map detailsMap = [:]
+        Map<String, Object> detailsMap = [:]
         List multiplePdList = []
```
Comment (croqueGrec09) [inline app/grails-app/services/de/laser/PropertyService.groovy:116]: Type of List?

File: app/grails-app/services/de/laser/TaskService.groovy [hunk 4]
```diff
@@ -21,17 +21,17 @@ class TaskService {
...
-            def query
+            String query
             if (flag == WITHOUT_TENANT_ONLY) {
                 query = select_with_join + 'where t.creator = :user and ru is null and t.responsibleOrg is null'
             } else {
                 query = select_with_join + 'where t.creator = :user'
             }
 
-            def params = [user : user]
+            Map params = [user : user]
```
Comment (croqueGrec09) [inline app/grails-app/services/de/laser/TaskService.groovy:34]: Map<String, Object>?

File: app/grails-app/services/de/laser/TaskService.groovy [hunk 5]
```diff
@@ -85,36 +85,38 @@ class TaskService {
...
+    List<Task> getTasksByResponsibles(User user, Org org, Map queryMap) {
+        List<Task> tasks = []
 
         if (user && org) {
-            def query = select_with_join + 'where ( ru = :user or t.responsibleOrg = :org ) ' + queryMap.query
+            String query = select_with_join + 'where ( ru = :user or t.responsibleOrg = :org ) ' + queryMap.query
             query = addDefaultOrder("t", query)
 
-            def params = [user : user, org: org] << queryMap.queryParams
+            Map params = [user : user, org: org] << queryMap.queryParams
```
Comment (croqueGrec09) [inline app/grails-app/services/de/laser/TaskService.groovy:119]: Map<String,Object>?

File: app/grails-app/services/de/laser/TaskService.groovy [hunk 6]
```diff
@@ -291,7 +291,7 @@ class TaskService {
         if (contextOrg) {
             if (binKonsortium) {
 
-                def qry_params_for_sub = [
+                Map qry_params_for_sub = [
```
Comment (croqueGrec09) [inline app/grails-app/services/de/laser/TaskService.groovy:294]: Map<String,Object>?

**Review by croqueGrec09 — COMMENTED** [review]
questions

File: app/grails-app/services/de/laser/PropertyService.groovy (hunk 2 above)
Comment (klodav) [reply app/grails-app/services/de/laser/PropertyService.groovy:114]: @return [<List<Long>, Map<String, Object>, List<Long>]

File: app/grails-app/services/de/laser/LicenseService.groovy (hunk 1 above)
Comment (klodav) [reply app/grails-app/services/de/laser/LicenseService.groovy:17]: returns [String, Map\<String, Object>]

**Review by croqueGrec09 — APPROVED** [review]
approved

**Pull request merged**